extensions/amp-story/1.0/test/test-amp-story-page.js

```diff
@@ -713,4 +714,69 @@ describes.realWin('amp-story-page', {amp: {extensions}}, (env) => {
 
     expect(startMeasuringStub).to.not.have.been.called;
   });
+
+  describe('maybeConvertCtaLayerToPageOutlink_', () => {
+    it('should do nothing if amp-story-cta-layer has two anchor tags', () => {
+      element.appendChild(
```

Comment:
You can run `appendChild` on `page.element` and then call `page.buildCallback`, you don't need to re-create a new `new AmpStoryPage`

Reply:
Can you please update the other tests?